server/svix-server/src/lib.rs

```diff
@@ -62,11 +62,11 @@ pub async fn run(cfg: Configuration, listener: Option<TcpListener>) {
         match cfg.queue_type {
             QueueType::Redis => {
                 let pool = crate::redis::new_redis_pool(redis_dsn()).await;
-                queue::redis::new_pair(pool).await
+                queue::redis::new_pair(pool, Duration::from_secs(45)).await
```

Comment:
Is this not a constant only for the testing? If so, I think it's better to just have a private module function that lets you fetch ignoring with a specific duration rather than having it always be set module-wide.

You don't usually want to uglify APIs just for tests. You want to have internal hooks.